app/javascript/react/components/LocationShowPage.js

```diff
@@ -48,11 +51,12 @@ const LocationShowPage = props => {
         newLocation.password_protected = password_protected_options[json.location.password_protected]
```

Comment:
I realize that lines 35 through 50 have already been merged. But upon further review I noticed that the flow control you currently have in the above lines should have already been managed on the backend, ideally in a serializer. If the `if` statements relied on some current state or props this might be different, but given that you are just checking the returned data, we these items should already be sorted prior to the info reaching your front end.